src/coreclr/vm/assembly.hpp

```diff
@@ -429,6 +521,14 @@ class Assembly
     BOOL                  m_fTerminated;
 
     LOADERHANDLE          m_hExposedObject;
+
+    // Load state tracking
+    FileLoadLevel   m_level;
+    BOOL            m_loading;
+    DWORD           m_notifyFlags;
+    Exception*      m_pError;
+    BOOL            m_fHostAssemblyPublished;
+    BOOL            m_bDisableActivationCheck;
```

Comment:
I would be nice to use lower-case bools where possible (more modern and space efficient).